internal/cmd/controller/cleanup/root.go

```diff
@@ -0,0 +1,38 @@
+package cleanup
+
+import (
+	"fmt"
+
+	command "github.com/rancher/fleet/internal/cmd"
+	"github.com/rancher/fleet/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+type CleanUp struct {
+	command.DebugConfig
```

Comment:
Doesn't `FleetManager` already includes a `DebugConfig`? Since it uses `PersistentPreRun`, I believe this is not needed.

Reply:
you're right! It's not needed to add it again in the subcomand. Removed in https://github.com/rancher/fleet/pull/1885/commits/8b1d535b72856e7501dd25512d867efcc9a5fe50